my_cinema_project/cinema/migrations/0003_alter_movie_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2 on 2024-02-12 15:33
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0002_rename_cinema_movie"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="movie",
+            options={"verbose_name_plural": "Movies"},
+        ),
+    ]
```

Comment:
It is a good practice for Django projects to include one migration per app per PR to not overload the `migrations` folder. Include only one migration to the project.